.github/workflows/build.yml

```diff
@@ -130,6 +130,12 @@ jobs:
             -   name: Validate database schema
                 run: (cd tests/Application && bin/console doctrine:schema:validate)
 
+            -   name: Run PHPStan
+                run: vendor/bin/phpstan analyse -c vendor/bitbag/coding-standard/phpstan.neon -l 8 src/
+
+            -   name: Run ECS
+                run: vendor/bin/ecs check src
```

Comment:
I would add `tests/Behat` and `spec` directories

Reply:
Added :D